packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -171,51 +170,68 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
 
     logger.info(`Found a precompiled version for ${opts.nativeModuleName} ${label}, unpacking.`);
 
-    tar
-      .x({
-        cwd: os.tmpdir(),
-        file: precompiledTarGzPath
-      })
-      .then(() => {
-        // See below for the reason why we append 'precompiled' to the path.
-        const targetDir = path.join(opts.nativeModulePath, 'precompiled');
+    let tar;
+
+    // tar 6.x supports Node >= 10
+    // Ensure we don't bring down customers application
+    try {
+      tar = require('tar');
+    } catch (err) {
```

Comment:
*question:* How about trying to require and then catching the error we explicitly check for the Node.js version and only attempt to require tar if we are on a supported Node.js version? The try-catch way probably works as well, I am not sure which one is actually better. WDYT?